consumer/pact-jvm-consumer-junit5/src/main/kotlin/au/com/dius/pact/consumer/junit5/PactConsumerTestExt.kt

```diff
@@ -308,35 +310,38 @@ class PactConsumerTestExt : Extension, BeforeTestExecutionCallback, BeforeAllCal
     if (!context.executionException.isPresent) {
       val store = context.getStore(NAMESPACE)
       val providerInfo = store["providerInfo"] as ProviderInfo
-      val pactDirectory = lookupPactDirectory(context)
       if (providerInfo.providerType != ProviderType.ASYNCH) {
```

Comment:
I realize the original code had the current order, but for clarity, I might switch the logic putting ProviderType.ASYNCH first...

```
override fun afterTestExecution(context: ExtensionContext) {
    if (!context.executionException.isPresent) {
      val store = context.getStore(NAMESPACE)
      val providerInfo = store["providerInfo"] as ProviderInfo
      if (providerInfo.providerType == ProviderType.ASYNCH) {
        storePactForWrite(store)
      } else {
        val mockServer = store["mockServer"] as JUnit5MockServerSupport
        Thread.sleep(100) // give the mock server some time to have consistent state
        mockServer.close()
        val result = mockServer.validateMockServerState(null)
        if (result is PactVerificationResult.Ok) {
          storePactForWrite(store)
        } else {
          JUnitTestSupport.validateMockServerResult(result)
        }
      }
    }
}
```

Reply:
Change made.